packages/app/src/cli/services/import-extensions.ts

```diff
@@ -6,17 +6,43 @@ import {ExtensionRegistration} from '../api/graphql/all_app_extension_registrati
 import {DeveloperPlatformClient} from '../utilities/developer-platform-client.js'
 import {MAX_EXTENSION_HANDLE_LENGTH} from '../models/extensions/schemas.js'
 import {OrganizationApp} from '../models/organization.js'
+import {allMigrationChoices} from '../prompts/import-extensions.js'
 import {renderSelectPrompt, renderSuccess} from '@shopify/cli-kit/node/ui'
 import {basename, joinPath} from '@shopify/cli-kit/node/path'
 import {writeFile} from '@shopify/cli-kit/node/fs'
 import {outputContent} from '@shopify/cli-kit/node/output'
 import {slugify} from '@shopify/cli-kit/common/string'
+import {AbortError} from '@shopify/cli-kit/node/error'
+
+const allExtensionTypes = allMigrationChoices.flatMap((choice) => choice.extensionTypes)
+
+export async function pendingExtensions(
+  remoteApp: OrganizationApp,
+  developerPlatformClient: DeveloperPlatformClient,
+  extensionTypes: string[] = allExtensionTypes,
+) {
+  const initialRemoteExtensions = await developerPlatformClient.appExtensionRegistrations({
+    id: remoteApp.apiKey,
+    apiKey: remoteApp.apiKey,
+    organizationId: remoteApp.organizationId,
+  })
+  const {extensionRegistrations} = initialRemoteExtensions.app
+  const extensions = await getExtensions({
```

Comment:
You are calling `appExtensionsRegistrations` twice no? one here and one inside `getExtensions`. Is that intentional? can we simplify it?